css/style.css

```diff
@@ -15,5 +18,247 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.karma_colone{
+  font-size: 70px;
+}
+hr{
+  width: 95%;
+}
+
+.nav_ul a:hover {
+  background-color: rgb(244, 29, 29);
```

Comment:
[The design](https://www.figma.com/file/BPtLAPr1HhwQdNfxGyFouo/Karma-design?node-id=23%3A0) shows orange text for the nav link hover effect, rather than a red background.